galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.cemt.manager.ivt/src/main/java/dev/galasa/cicsts/cemt/manager/ivt/CEMTManagerIVT.java

```diff
@@ -80,13 +82,13 @@ public void testCEMTIsNotNull() throws CicstsManagerException {
    
    @Test
    public void testInquireResource() throws InterruptedException, CicstsManagerException{
-      assertThat(cics.cemt().inquireResource(cemtTerminal, "PROGRAM", "EXAMPLE").get("program").equals("EXAMPLE")); 
+	   assertThat(cics.cemt().inquireResource(cemtTerminal, "PROGRAM", "EXAMPLE").get("program").equals("EXAMPLE")); 
```

Comment:
would like to see more fields in the inquire be tested

Reply:
I have merged this with another test method that tested all fields of the hash map returned by the inquire method, hopefully this suffices now.

Reply:
OK Will check